guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc

```diff
@@ -45,7 +45,14 @@ Do not specify a passphrase.
 ----
 # sudo --user foreman ssh _git.example.com_
 ----
-. Configure the Template Sync plugin settings on a *Template Sync* tab.
-.. Change the *Branch* setting to match the target branch on a Git server.
-.. Change the *Repo* setting to match the Git repository.
-For example, for the repository located in `git@git.example.com/templates.git` set the setting into `git@git.example.com/templates.git`.
+. In the {ProjectWebUI}, provide details for your Git repository:
+.. Navigate to *Administer > Settings*.
+.. Click the *Template Sync* tab.
+.. Change the *Branch* setting to define the target branch of the repository.
+.. Change the *Repo* setting to define the repository URL.
+.. If you want to use an HTTP proxy to connect to the repository, change the *HTTP proxy policy* setting to define your HTTP proxy policy.
```

Comment:
Maybe something like this would add a bit more clarity? Although I am not certain if I am not being overly specific.
```suggestion
.. If you want to use an HTTP proxy to connect to the repository, change the *HTTP proxy policy* setting to `Global default HTTP proxy` or `Custom HTTP proxy`.
If you select `Custom HTTP proxy`, you will be prompted to select the HTTP proxy when syncing templates.
```

Reply:
I think that considering what the web UI will say, we can go with only the first of the two lines you're suggesting. Thanks!